.env.example

```diff
@@ -0,0 +1,3 @@
+DJANGO_SECRET_KEY=django-insecure-0vesy_dd%v&m*y%59xng_%h3a-f7^=)dqns2u$%$mp$g@wy^&c
+DEBUG=True
+ALLOWED_HOSTS=127.0.0.1
```

Comment:
We don't have to fill out any value in .env.example file. Values should be empty.